README.md

```diff
@@ -0,0 +1,96 @@
+# search-v3
+
+This PHP library allows you to integrate with publiq's Search API3.
+
+Full API documentation: https://documentatie.uitdatabank.be/content/search_api_3/latest/start.html <br />
```

Comment:
I wonder if a `<br />` is handled in markdown?

Reply:
Yes https://www.markdownguide.org/basic-syntax/#line-breaks

Reply:
> To create a line break (<br>), end a line with two or more spaces, and then type return.
The table show the `br` inside the HTML column?

Reply:
> Line Break Best Practices
>
> You can use two or more spaces (commonly referred to as “trailing whitespace”) for line breaks in nearly every Markdown application, but it’s controversial. It’s hard to see trailing whitespace in an editor, and many people accidentally or intentionally put two spaces after every sentence. For this reason, you may want to use something other than trailing whitespace for line breaks. Fortunately, there is another option supported by nearly every Markdown application: the <br> HTML tag.

Reply:
PHPStorm also strips trailing whitespaces

Reply:
Github renders it correctly btw https://github.com/cultuurnet/search-v3/tree/rc-1.0